src/main/java/jackbean/command/Parser.java

```diff
@@ -9,112 +9,113 @@
 import jackbean.task.Todo;
 
 public class Parser {
-    public static String parseUserInput(String input, TaskList taskList) throws NotEnoughArgumentsException, TooManyArgumentsException, InvalidTaskTypeException {
-            if (input.equalsIgnoreCase("help")) {
-                // GitHub copilot helped mostly with the response to help message
-                StringBuilder reply = new StringBuilder("Yo homie! Here are the commands you can use:\n");
-                reply.append("1. list - to list all tasks\n");
-                reply.append("2. todo <description> - to add a todo task\n");
-                reply.append("3. deadline <description> /by <date> - to add a deadline task\n");
-                reply.append("4. event <description> /from <date> /to <date> - to add an event task\n");
-                reply.append("5. mark <task number> - to mark a task as done\n");
-                reply.append("6. unmark <task number> - to mark a task as undone\n");
-                reply.append("7. bye - to exit the program\n");
-                reply.append("8. help - to see this message again\n");
-                reply.append("9. delete <task number> - to delete a task\n");
-                reply.append("10. maybe some easter eggs in the future? :)");
-                return reply.toString();
-            } else if (input.equalsIgnoreCase("list")) {
-                StringBuilder reply = new StringBuilder("Yo homie!, here are the tasks in your list:");
-                for (int i = 1; i <= taskList.getSize(); i++) {
-                    reply.append("\n").append(i).append(". ").append(taskList.getTask(i));
-                }
-                return reply.toString();
-            } else if (input.toLowerCase().startsWith("mark")) {
-                // first use split and then parse the integer
-                String[] splitInput = input.split(" ");
-                int taskNumber = Integer.parseInt(splitInput[1]);
+    public static String parseUserInput(String input, TaskList taskList)
```

Comment:
Maybe can consider adding java docs for your public methods!

Reply:
Sure thing!